src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final String[] DEFAULT_TITLES = {"fruit", "quantity"};
+    private static Map<String, Integer> fruits = new HashMap<>();
+
+    public static void change(String fruit, Integer amount) {
+        fruits.put(fruit,Storage.fruits.get(fruit) + amount);
+    }
+
+    public static Map<String, Integer> getFruits() {
+        return fruits;
+    }
```

Comment:
storage should only store the current amount of some fruit, nothing else
just make the map public, and remove everything else